extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/filters/accesslog/DefaultAccessLogReceiver.java

```diff
@@ -226,38 +173,13 @@ public void run() {
         }
     }
 
-    /**
-     * For tests only. Blocks the current thread until all messages are written
-     * Just does a busy wait.
-     * <p/>
-     * DO NOT USE THIS OUTSIDE OF A TEST
-     */
-    void awaitWrittenForTest() throws InterruptedException {
-        while (!pendingMessages.isEmpty() || forceLogRotation) {
-            Thread.sleep(10);
-        }
-        while (state != 0) {
-            Thread.sleep(10);
-        }
-    }
-
     private void writeMessage(final List<String> messages) {
-        if (System.currentTimeMillis() > changeOverPoint) {
+        if (now().toEpochMilli() > changeOverPoint) {
```

Comment:
changed for testing, but without success. had to apply reflection whitebox testing -1.

- https://www.baeldung.com/java-override-system-time